### PR TITLE
Set #default_options in subscriptions requests.

### PR DESCRIPTION
**Pull request by frodsan**

### CODE REVIEW

File: lib/facebook/messenger/subscriptions.rb [hunk 1]
```diff
@@ -8,30 +8,37 @@ module Subscriptions
...
       end
 
+      def default_options
+        super.merge(
+          format: :json,
+          query: {
+            access_token: Facebook::Messenger.config.access_token
+          }
+        )
+      end
```
Comment (jgorset) [inline lib/facebook/messenger/subscriptions.rb]: The tricky thing about `HTTParty.default_options` is that it's evaluated once at compile time and cached forever. Our access tokens don't change, though, so I guess it works in this case (but a good thing to be aware of).

File: lib/facebook/messenger/subscriptions.rb [hunk 2]
```diff
@@ -8,30 +8,37 @@ module Subscriptions
...
       end
 
-      def self.raise_errors(response)
+      def raise_errors(response)
         raise Error, response['error']['message'] if response.key? 'error'
       end
 
+      def default_options
+        super.merge(
+          format: :json,
```
Comment (jgorset) [inline lib/facebook/messenger/subscriptions.rb]: I think we should set this as `format :json` instead, like we've done for `base_uri`.

File: lib/facebook/messenger/subscriptions.rb (hunk 1 above)
Comment (frodsan) [inline lib/facebook/messenger/subscriptions.rb]: 👍

**Comment by jgorset** [comment]
I'm running out of :cake: here!

**Pull request merged**